torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -554,6 +554,18 @@ void Reducer::delay_all_reduce() {
     }
   }
 
+  if (ddp_debug_level_ == c10d::DebugLevel::Info || ddp_debug_level_ == c10d::DebugLevel::Detail) {
```

Comment:
Add a comment explaining why we are logging it?

Reply:
Yes, will do.

Reply:
Comments added, please help review again, thank you @rohan-varma !